README.md

```diff
@@ -5,7 +5,12 @@ Extra manifests for [Scoop](https://scoop.sh), the Windows command-line installe
 To make it easier to install apps from this bucket, run
     `scoop bucket add extras`
 
-## contributing
-For documentation on how to add packages to the Extras bucket see [the Wiki](https://github.com/ScoopInstaller/Scoop/wiki) from main [Scoop](https://github.com/ScoopInstaller/Scoop) project, particularly: 
+## How do I install these manifests?
+
+To add this bucket, run `scoop bucket add extras`. To install, do `scoop install <manifest>`.
```

Comment:
This repeats the same instruction found on lines 5-6, but your verbiage is better. I would love to see us have standard verbiage across all the buckets we maintain. Perhaps the verbiage could live in https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md as well.